includes/import.form.inc

```diff
@@ -967,6 +975,31 @@ function islandora_multi_importer_objectmapping_form($form, &$form_state, $file_
         '#markup' => '&nbsp;',
       ),
     ),
+
+    /* MAM. Added to support issue #20...but this element currently does NOTHING so it's been commented out.
```

Comment:
OK, I actually prefer you uncomment this and i make a second pull to actually make this work fieldwork once this is merged. Sounds good?